.jshintrc

```diff
@@ -25,7 +25,7 @@
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
   "maxparams"     : 10,       // {int} Max number of formal params allowed per function
```

Comment:
And this is what I was referring to earlier -
`"maxparams"     : 10`
Most of our `ngInjects` are dangerously close to 10